docs/content/en/samples/testers/custom/customTest.yaml

```diff
@@ -0,0 +1,12 @@
+test:
+  - image: custom-test-example
+    custom:
+      - command: ./test.sh
+        timeoutSeconds: 60
+        dependencies:
+          paths:
+          -  "*_test.go"
+          -  "test.sh"
+      - command: echo Hello world!!
+        dependencies:
+          command: echo [\"main_test.go\"]
```

Comment:
add newline character at the end on all files that don't

Reply:
Added new line.